### PR TITLE
policy-engine: set custom buckets for v1_graph_serve_duration_seconds

### PR DESCRIPTION
**Pull request by vrutkovs**
Set custom buckets based on metrics from August

### CODE REVIEW

File: policy-engine/src/graph.rs [hunk 1]
```diff
@@ -7,20 +7,27 @@ use cincinnati::CONTENT_TYPE;
 use commons::{self, GraphError};
 use failure::Fallible;
 use futures::{future, Future};
-use prometheus::{Counter, Histogram, HistogramOpts, Registry};
+use prometheus::{Counter, Histogram, histogram_opts, Registry};
 use serde_json;
 use std::collections::HashMap;
 
+pub const GRAPH_SERVE_BUCKETS: &[f64; 11] = &[
```
Comment (steveej) [inline policy-engine/src/graph.rs]: A docstring with a rationale and a short explanation of the values would be nice.

**Review by steveej — CHANGES_REQUESTED** [review]

File: policy-engine/src/graph.rs (hunk 1 above)
Comment (vrutkovs) [reply policy-engine/src/graph.rs]: Done, PTAL

Comment (steveej) [reply policy-engine/src/graph.rs]: Thanks! I was hoping for a description on the values, but I realize that I simply have to read the prometheus docs for their meaning.

File: policy-engine/src/graph.rs [hunk 2]
```diff
@@ -7,19 +7,25 @@ use cincinnati::CONTENT_TYPE;
...
         "v1_graph_incoming_requests_total",
         "Total number of incoming HTTP client request to /v1/graph"
     )
     .unwrap();
-    static ref V1_GRAPH_SERVE_HIST: Histogram = Histogram::with_opts(HistogramOpts::new(
+    static ref V1_GRAPH_SERVE_HIST: Histogram = Histogram::with_opts(histogram_opts!(
         "v1_graph_serve_duration_seconds",
-        "HTTP graph serving latency in seconds"
+        "HTTP graph serving latency in seconds",
+        GRAPH_SERVE_BUCKETS.to_vec()
```
Comment (steveej) [inline policy-engine/src/graph.rs]: If this is the only place where `GRAPH_SERVE_BUCKETS` is consumed I'd inline the value.

**Review by steveej — APPROVED** [review]
In addition to the nitpick in the comments, there's also the commit message which should say `policy-engine: ...`. The term _graph_engine_ isn't used anywhere AFAIK.

**Review by steveej — CHANGES_REQUESTED** [review]
Accidentally approved. Please see the comments from the previous review

File: policy-engine/src/graph.rs (hunk 2 above)
Comment (vrutkovs) [reply policy-engine/src/graph.rs]: Done, it makes comment a bit ugly though

**Comment by vrutkovs** [comment]
```
+ rmdir /srv/volume/jenkins/workspace/openshift-cincinnati-gh-pr-check@2/dist/..//release-20190913.113637
rmdir: failed to remove '/srv/volume/jenkins/workspace/openshift-cincinnati-gh-pr-check@2/dist/..//release-20190913.113637': No such file or directory
Build step 'Execute shell' marked build as failure
```
/retest

**Pull request merged**